cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -0,0 +1,38 @@
+
+{# ==========================================================================
+
+   table.render()
+
+   ==========================================================================
+
+   Description:
+
+   Create a gray well organism.
+   See [GHE]/flapjack/Modules-V1/wiki/Well
+
+   table.row
+
+
+   ========================================================================== #}
+
+{% macro render(thead, rows) %}
+<table class="table__stack-on-small table__entry-header-on-small">
```

Comment:
It's not included in [cf-tables](https://cfpb.github.io/cf-tables/demo/) and I think something like this on BEM style should be included in our front-end standards if we have a standard.